### DISCUSSION TITLE
Player average fantasy points?

### DISCUSSION THREAD

**Discussion by TPassalacqua** [cwendt94/espn-api#476] [ESPN API] [answered]
Can't find an endpoint for average fantasy points. Is this possible without checking each week and doing the math on the backend?

**Comment by TPassalacqua** [comment]
Still haven't found any average_points datapoint. However it seems a viable way to do this is parsing the stats output for games played (this seems to be the "210" key) and total points then calculating the avg.

**Comment by cwendt94** [comment]
Thats a good question, I believe the stats returned from the player info has that information. I should be able to pull that out and add a new field in the player class for average points. We are currently getting it for [basketball](https://github.com/cwendt94/espn-api/wiki/Player-Class-Basketball) and should be able to do the same for football

**Reply by cwendt94** [reply, marked as answer]
Avg points have been added to the [player class](https://github.com/cwendt94/espn-api/wiki/Player-Class) in the new version of the package v0.33.0
